src/cli/commands/create/app.ts

```diff
@@ -419,7 +419,7 @@ async function helloWorldTemplate(layout: Layout.Layout) {
         t.list.field('worlds', {
           type: 'World',
           resolve(_root, _args, ctx) {
-            return ctx.db.worlds
+            return ctx.memoryDB.worlds
```

Comment:
This change was made so that when adding the prisma plugin the context db key that it adds will not conflict with this scaffolded app.